get-started.md

```diff
@@ -2,50 +2,73 @@
+In addition, the other sections on this page contain pointers to other useful sources of information for 2i2c.
 
-Someone should have sent you a message with your 2i2c email address. You can use this to access information in the 2i2c drive, as well as various online services that we use such as cloud engines. You can sign in to your email address at gmail.com (it behaves like a Google account).
+## Onboarding process
 
-### Google Drive
+New 2i2c team members are onboarded in order to make sure that the following conditions are met:
 
-Your 2i2c email address will get you access to a Google Drive that contains documents, brainstorms, and other assets that aren't fit for the 2i2c Team Compass. You can log into the Google Drive by going to `drive.google.com` and adding a new account using your `@2i2c.org` address and password.
+- They know where to look for certain information
+- They feel welcome and excited to be a part of the team
+- They have an understanding of what 2i2c is currently up to
+- They have enough information to get started
 
-### GitHub Access
+When a new team member joins, follow these two steps first:
 
-2i2c has a few GitHub repositories. You should have membership in the `2i2c-org` GitHub organization and write access to its repositories. Check that this is the case and, if not, ask for access in the Slack.
+1. Identify an **Onboarding Champion**. This is somebody that will help guide the onboarding process for our new team member.
+   Their job is to carry out the process described in the onboarding issue template.
+2. [Open an issue in the Team Compass](https://github.com/2i2c-org/team-compass/issues/new/choose). Choose the "Onboard a new team member" template.
+   This issue will track the onboarding process, and will provide steps to take in order to onboard our new team member.
 
+The following sections describe some things that should be met in order to onboard somebody new.
 
-### NameCheap Access
+## Accounts that should be created
 
-2i2c runs much of its cloud infrastructure at custom URLs created for the target institution. In order to ensure you have the ability to create new URLs, you should have access to the 2i2c NameCheap account. If not, ask for access in the Slack.
+- **A GMail address**. You can use this to access information in the 2i2c drive, as well as various online services that we use such as cloud engines. You can sign in to your email address at gmail.com (it behaves like a Google account).
+- **A Slack account**. In addition to GMail, 2i2c also uses Slack to have informal or synchronous communication.
 
+## Accounts you should have access to
 
-### Cloud Accounts
+- **Google Drive**. Your 2i2c email address will get you access to a Google Drive that contains documents, brainstorms, and other assets that aren't fit for the 2i2c Team Compass. You can log into the Google Drive by going to `drive.google.com` and adding a new account using your `@2i2c.org` address and password.
```

Comment:
Actually, you needed to give me access to the 2i2c folder, otherwise, the drive is empty for the onboardee.